corehq/form_processor/interfaces/dbaccessors.py

```diff
@@ -382,8 +382,12 @@ def get_closed_case_ids_for_owner(self, owner_id):
     def get_all_reverse_indices_info(self, case_ids):
         return self.db_accessor.get_all_reverse_indices_info(self.domain, case_ids)
 
-    def get_reverse_indexed_cases(self, case_ids):
-        return self.db_accessor.get_reverse_indexed_cases(self.domain, case_ids)
+    def get_reverse_indexed_cases(self, case_ids, case_types=None, is_closed=None):
+        # TODO only the SQL version accepts these args, how should we handle?
+        # assert is_sql or case_types is None and is_closed is None
+        # Mimic the functionality in memory for couch?
+        # Change couch view and reindex?
```

Comment:
This PR changes the method signature for only the SQL version.  Do we have a convention for how to handle that?  I laid out some options here.  I don't think this has much utility outside of custom dev, so the couch version may never get used.

Reply:
@snopoke Do you have any suggestions for resolving this?

Reply:
I think that changing couch views on forms/cases would take a long time to reindex.

It seems reasonable to do it in memory for couch. That's what it's doing now anyways right? If so then there shouldn't be any negative effects